RealmJS.podspec

```diff
@@ -0,0 +1,97 @@
+require 'json'
+
+package = JSON.parse(File.read(File.expand_path('package.json', __dir__)))
+
+# Building the podspec from a local .git repository when developing - GitHub otherwise
+git_path_local = File.expand_path('.git', __dir__)
+git_path = File.directory?(git_path_local) ? "file://#{git_path_local}" : 'https://github.com/realm/realm-js.git'
```

Comment:
I don't think this does anything? The Spec source path is used to find out where to get the code when installing from the repository. Unless RN is doing something very strange it's not applicable to local installs at all, and normal usage of Cocoapods doesn't require changing it for local testing of pods.

Reply:
I realized this way too late in the process. I had been testing the pod with `pod spec lint`, but as React Native loads in the pod from its local path, I just need the podspec to pass `pod lib lint`. I've pushed a change to simplify this.